src/main/java/net/minecraftforge/common/config/ConfigManager.java

```diff
@@ -161,10 +162,132 @@ public static void load(String modid, Config.Type type)
             }
         }
     }
+    
+    public static void updateConfig(String modid, Config.Type type)
+    {
+        FMLLog.fine("Attempting to update Configuration objects of mod %s for type %s", modid, type);
```

Comment:
Again, who is calling this, and why are they updating all of the specific of the type at once?